pkg/email/lore/parse.go

```diff
@@ -20,9 +22,29 @@ type Thread struct {
 	Messages  []*email.Email
 }
 
+type Series struct {
+	Subject   string
```

Comment:
Does it happen that the subject changes between versions?
If so, do we need to also extract in-reply to message-id, which may be the previous version?

Reply:
AFAIK the series are usually not sent in-reply-to the previous ones, so I'd expect the title to be fairly stable.